eslintrc.js

```diff
@@ -0,0 +1,158 @@
+module.exports = {
```

Comment:
And this file should be removed also

Reply:
Hey, @LotusElize, please revert this file to its original state as you've done for `.gitignore`

Reply:
Your changes should be done in your folder only. Now it will change the `.eslint.js` for the whole project, if I merge your pr